drivers/eks/eks_driver.go

```diff
@@ -1289,6 +1315,87 @@ func (d *Driver) SetVersion(ctx context.Context, info *types.ClusterInfo, versio
 	return nil
 }
 
+func (d *Driver) updateNodePoolAndAwait(state state, info *types.ClusterInfo) error {
```

Comment:
I think the receiver can be removed
```suggestion
func updateNodePoolAndAwait(state state, info *types.ClusterInfo) error {
```